src/tools/MappingTools.ts

```diff
@@ -23,6 +23,51 @@ import { EnumDeclaration } from '../components/general/EnumDeclaration';
 import { EnumElement } from '../components/general/EnumElement';
 import { SyntaxKind } from 'typescript';
 
+
+function setExtractedObjectValues(readObject, extractedObject, sourceFile: ts.SourceFile): void {
```

Comment:
This is totally great! Just one small thing, could you move this method below `mapObjectLiteral(...)`.

I think that it is more readable to have `mapFile(sourceFile: ts.SourceFile)` method as the first one, because it is the main entry point to this class. Thanks!